lib/bundler/cli/install.rb

```diff
@@ -185,5 +181,13 @@ def print_post_install_message(name, msg)
       Bundler.ui.confirm "Post-install message from #{name}:"
       Bundler.ui.info msg
     end
+
+    def check_for_options_conflicts
```

Comment:
`raise` something like an `InstallError` instead of exiting? in the rest of the `run` method, `exit 1` is used, though. Thoughts?